localstack/services/awslambda/lambda_api.py

```diff
@@ -363,8 +361,7 @@ def process_apigateway_invocation(
         event["pathParameters"] = path_params
         event["resource"] = resource_path
         event["requestContext"] = request_context
-        if stage_variables:
-            event["stageVariables"] = stage_variables
+        event["stageVariables"] = stage_variables
```

Comment:
```suggestion
        event["stageVariables"] = stage_variables or {}
```

Not sure `stage_variables` is ever None but either this change or keeping the if statement in L352 would at least keep the previously intended behavior.

Reply:
@dominikschubert the behaviour still the same even without your suggestion since the assignment is done from `get_stage_variables` which returns `{}` or None. AWS docs point to `null` when there are no values, but I think we don't follow exactly the same pattern everywhere.